### ISSUE TITLE
Problems with displaying Schema composition (allOf)

### ISSUE THREAD

**Issue by GitGraf** [mrin9/RapiDoc#97]
RapiDoc-Version: 7.0.4
I have the following example OpenAPI spec:
```yml
openapi: 3.0.0
info:
  title: My OpenAPI Spec
  version: v1
paths:
  /baseEndpoint:
    get:
      tags: 
        - openapi
      summary: List some stuff
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                type: object
                $ref: '#/components/schemas/BaseObject'
    post: 
      tags:
        - openapi
      requestBody:
          content:
            application/json:
              schema:
                type: object
                $ref: '#/components/schemas/BaseObject'
  /advancedEndpoint:
    get:
      tags: 
        - openapi
      summary: List some advanced stuff
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                type: object
                $ref: '#/components/schemas/ExtendedObject'
    post: 
      tags:
        - openapi
      requestBody:
          content:
            application/json:
              schema:
                type: object
                $ref: '#/components/schemas/ExtendedObject'
components:
  schemas:
  
    BaseObject:
      type: object
      properties: 
        baseProperty:
          type: integer
      example: 
        baseProperty: 1234
        
    ExtendedObject:
      type: object
      # The problem seems to be here. If "allOf" is removed and the properties of "BaseObject" are manually copied, the schema is displayed as expected.
      allOf:
        - $ref: '#/components/schemas/BaseObject'
        - properties: 
            extendedProperty:
              type: string
      example:
        extendedProperty: 'hello'
        
tags:
  - name: openapi
    description: My awesome Endpoint Tag
```
The problem is that the Models for the "extendedEndpoint"s don't appear to be rendered. If the `allOf` keyword is removed and the properties are manually copied, then the Models are rendered correctly (See attached Screenshots).

The provided specification seems to be rendered correctly over at editor.swagger.io.

Is `allOf` currently supported by RapiDoc?

![rapidoc_ok](https://user-images.githubusercontent.com/19150483/69959107-9ad72700-1506-11ea-882b-f647ecab5f2b.PNG)
![rapidoc_allOf](https://user-images.githubusercontent.com/19150483/69959111-9d398100-1506-11ea-866b-346794e58cff.PNG)

**Comment by mrin9** [comment]
may be when you put `all-of` at the top-level in schema definition it gets into an issue.
thats my quick analysis, can you confirm if thats the case? coz the following example using `all-of` works .

```
openapi: 3.0.0
info:
  title: all-of example
  description: Model composition using all-of
  version: 1.0.0
paths:
  /api/employee:
    get:
      summary: List of employees
      responses:
        "200":
          description: Checkout the model tab to view object schemas defined using all-of
          content:
            application/json:
              schema:
                type: array
                items:
                  $ref: "#/components/schemas/employee"
components:
  schemas:
    employee:
      type: object
      additionalProperties: false
      properties:
        Employee:
          description: Employee Details
          allOf:
            - properties:
                Id:
                  type: string
                  description: Employee ID
            - $ref: "#/components/schemas/name"
            - $ref: "#/components/schemas/job"
    name:
      type: object
      additionalProperties: false
      properties:
        firstName:
          type: string
          description: First name
        lastName:
          type: string
          description: Last Name
    job:
      type: object
      additionalProperties: false
      properties:
        jobTitle:
          type: string
          description: Job title
        department:
          type: string
          description: Department name
```

**Comment by GitGraf** [comment]
After playing around with your provided specification for a while, it appears to me that your assumption is correct. Model composition only seems to be problematic when used at the top level of the schema. 

What I find to be interesting is that `allOf` seems to be working correctly for schemas defined under the `paths` property:

```yml
paths:
  /api/employee:
    get:
      summary: List of employees
      responses:
        "200":
          description: Checkout the model tab to view object schemas defined using all-of
          content:
            application/json:
              schema:
                type: array
                items:
                  # Seems to have the desired effect, even though we are technically at the Top-Level of the schema here (?)
                  allOf: 
                    - $ref: "#/components/schemas/name"
                    - $ref: "#/components/schemas/job"
                    - properties: 
                        Id: 
                          type: string
                          description: Employee ID
    [...]
```

**Comment by mrin9** [comment]
thanks for confirming.
This is fixed in the commit  55398ff9dff9e19e8179ca0ff950e33aea5dcd4a

it is not yet released, to test it, you can either build it yourself, or use `rapidoc-min.js` from `dist` folder
here is one test case https://mrin9.github.io/RapiDoc/examples/allof.html
and the spec can be found here https://github.com/mrin9/RapiDoc/blob/master/docs/specs/allof.yaml

feel free to reopen, in case its not working as you expect it to be

> do not use `rapidoc-min.js` from `dist` folder in production, we do not actively support it. and can be removed or changed without any release

**Comment by GitGraf** [comment]
Thank you for the quick support on this issue! I will be trying out the fixed version some time this week. Cheers!
